pb-rs/README.md

```diff
@@ -41,7 +41,9 @@ fn main() {
         headers: false, // do not generate headers
         dont_use_cow: false, // Don't use Cow<_,_> for Strings and Bytes
         custom_struct_derive: vec![], // Nothing
-        custom_rpc_generator: RpcGeneratorFunction::default(),
+        custom_rpc_generator: pb_rs::types::RpcGeneratorFunction::default(),
```

Comment:
There was no default implementation for me on the latest release code. I had to use this to compile:

```suggestion
        custom_rpc_generator: Box::new(|_, _| Ok(())),
```